src/index.css

```diff
@@ -1,13 +1,137 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+.wrapper {
+  display: flex;
+  align-items: center;
+}
+
+h1 {
+  display: flex;
+  align-items: center;
+}
+
+.nav {
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 85px;
+  background-color: white;
+}
+
+.same {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.heading {
+  font-family: 'Times New Roman', Times, serif;
+}
+
+ul {
+  list-style: none;
+  padding-left: 0;
+}
+
+a {
+  color: #a4a7af;
+  text-decoration: none;
+  margin: 0 10px;
+  font-size: 15px;
+  cursor: pointer;
+}
+
+a:hover {
+  color: black;
+  cursor: pointer;
+  text-decoration: underline;
+}
+
+.rocketImage {
+  width: 200px;
+  height: 150px;
+  padding: 10px 20px;
+}
+
+.miss-table {
+  font-family: 'Merriweather', serif;
+  width: 95%;
+  margin: 2rem auto;
+}
+
+small {
+  background-color: #0290ff;
+  font-size: 18px;
+  margin-right: 5px;
+}
+
+.first-col {
+  padding: 1rem;
+  text-align: left;
+  font-size: 24px;
+}
+
+.second-col {
+  padding: 2rem 1rem;
+  font-size: 20px;
+  text-align: left;
+  line-height: 2;
+}
+
+.third-col {
+  text-align: center;
+  font-size: 26px;
+}
+
+.member {
+  background-color: rgb(13, 218, 13);
+  border-radius: 10px;
+  padding: 0.5rem;
+  color: white;
+}
```

Comment:
- [ ] Please let also improve the design of these badges. Try and match the provided design as much as possible.

| The provided design | Your implementation |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/45002654/147277277-f7ede7ea-3c68-448b-a5c2-4afcb5da6fa4.png) ![image](https://user-images.githubusercontent.com/45002654/147277455-90680adc-4acb-41d9-9ad8-cdb0003cd176.png) | ![image](https://user-images.githubusercontent.com/45002654/147277529-9e51b679-6d7d-4b9a-8b3a-24985ab89a52.png) ![image](https://user-images.githubusercontent.com/45002654/147277572-04b49211-4273-417e-8669-4937d54a3598.png) |

Try and match the background color and the size of the badges. Consider reducing their font sizes.

Reply:
Implemented.